jms/1.9/modules/ROOT/pages/jms-performance.adoc

```diff
@@ -76,6 +76,39 @@ In the XML editor, the `jms:topic-consumer`, `shared`, and `primaryNodeOnly` con
  </jms:listener>
 ----
 
+== Configure Prefetch for JMS Connector
+
+For JMS Connector, when using ActiveMQ client, you configure prefetch on the server connection URL. Refer to the https://activemq.apache.org/components/classic/documentation/what-is-the-prefetch-limit-for#:~:text=Specifying%20the%20PrefetchPolicy[ActiveMQ documentation] for details.
+
+For example, consider these connection URLs with the `jms.prefetchPolicy.all` parameter:
+[source,xml,linenums]
+----
+tcp://localhost:61616?jms.prefetchPolicy.all=50
+tcp://localhost:61616?jms.prefetchPolicy.queuePrefetch=1
+----
+
+In the JMS Connector XML configuration, the configuration of the connection `brokenUrl` looks like this:
+
+[source,xml,linenums]
+----
+<jms:config name="JMS_Config_listen" doc:name="JMS Config" doc:id="6600826c-0798-496b-b016-9b9b29268cb8" >
+		<jms:active-mq-connection username="admin" password="admin" >
+				<reconnection >
+				<reconnect-forever />
+			</reconnection>
+			<jms:caching-strategy >
+				<jms:no-caching />
+			</jms:caching-strategy>
+			<jms:factory-configuration brokerUrl="tcp://localhost:61616?jms.prefetchPolicy.all=1" enable-xa="true" />
+		</jms:active-mq-connection>
+		<expiration-policy maxIdleTime="30" timeUnit="SECONDS" />
+		<jms:consumer-config maxRedelivery="10" />
+		<jms:producer-config persistentDelivery="true" />
+	</jms:config>
+----
+
+Use a large prefetch values for high performance with high message volumes. However, for lower message volumes, where each message takes a long time to process, set the prefetch to `1`. This ensures that a consumer is only processing one message at a time. Specifying a prefetch limit of zero, however, causes the consumer to poll for messages, one at a time, instead of the message being pushed to the consumer.
```

Comment:
```suggestion
Use large prefetch values for high performance with high message volumes. However, for lower message volumes, where each message takes a long time to process, set the prefetch to `1`. This ensures that a consumer is only processing one message at a time. Specifying a prefetch limit of zero, however, causes the consumer to poll for messages, one at a time, instead of the message being pushed to the consumer.
```